ykman-gui/qml/ContentStack.qml

```diff
@@ -180,6 +180,10 @@ StackView {
                                    views.selectedSlot)).arg(snackbarError.getDefaultMessage(error)))
     }
 
+    function snackbarErrorMessage(error) {
+        snackbarError.show(qsTr(snackbarError.getDefaultMessage(error)))
```

Comment:
`qsTr` should only be applied to literals, not variable expressions. We can also just use the `snackbarError.showResponseError` method. :)
```suggestion
        snackbarError.showResponseError(error)
```

There are a bunch of preexisting misuses of `qsTr` like this, too, but let's take care of those in a separate PR.

Reply:
Will fix!